### DISCUSSION TITLE
PSDT (Partially Signed Dogecoin Transaction)

### DISCUSSION THREAD

**Discussion by xincognitodev** [dogecoin/dogecoin#2955] [General]
Hello Shibs,

I have been searching for PSBT support to DOGE hardware wallets but unfortunately none of the hard wallet service providers are come up with a solution for Doge. All I have found is PSBT support for Bitcoin(Cold wallets). Since we are growing in much rapid phase this feature (PSDT) will be very useful for the users who want to safely use Hardware/Cold Doge wallets.
Please share your thought on this (PSDT) development.

Appreciate your help!!

- To The Moon!!

**Comment by patricklodder** [comment]
Version 0 of PSBT ([BIP 174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki)) is fully usable for Dogecoin; we don't need to develop anything and can simply use the Bitcoin standard. 

The only requirement is to use the `PSBT_IN_NON_WITNESS_UTXO` field because there is no segwit in Dogecoin. This means that you need to provide the source transaction for each input that you include in your transaction, the same as you would for any non-segwit Bitcoin transaction.

_Edit: version 1 is actually called version 0_

**Reply by xincognitodev** [reply]
Do we have any tools like Electrum for [BIP‑44], [BIP‑49], [BIP‑84] support?

**Reply by patricklodder** [reply]
There is/was someone working on electrum, but I haven't heard from them in a while and am not sure how far along they are. See #2831

**Reply by xincognitodev** [reply]
Thankyou!!
 I will try to explore the option
